src/Apps/Artist/routes.tsx

```diff
@@ -155,18 +155,15 @@ export const routes: RouteConfig[] = [
           }
         `,
       },
-      // Redirect old Artist app links to Overview page
+      // Redirect all unhandled tabs to the artist page.
+      // Note: there is a deep-linked standalone auction-lot page
+      // in Force, under /artist/:artistID/auction-result/:id
+      // That app needs to be mounted before this app for that to work,
+      // and not get caught here.
       new Redirect({
-        from: "/works",
+        from: "/:other",
```

Comment:
So if I understand this correctly the `:other` is effectively a coerced wildcard, similar to the `*` below?

Reply:
Yea, exactly, that's why I removed the catch-all, this is the new catch-all essentially.

good call, will rename.